File: support_tools/swigwin-3.0.12/Examples/test-suite/go/arrays_global_runme.go
package main

import . "./arrays_global"

func main() {
	SetArray_i(GetArray_const_i())

	GetBeginString_FIX44a()
	GetBeginString_FIX44b()
	GetBeginString_FIX44c()
	GetBeginString_FIX44d()
	GetBeginString_FIX44d()
	SetBeginString_FIX44b("12\00045")
	GetBeginString_FIX44b()
	GetBeginString_FIX44d()
	GetBeginString_FIX44e()
	GetBeginString_FIX44f()

	Test_a("hello", "hi", "chello", "chi")

	Test_b("1234567", "hi")
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/arrays_global_runme.py
import arrays_global

arrays_global.cvar.array_i = arrays_global.cvar.array_const_i

from arrays_global import *

BeginString_FIX44a
cvar.BeginString_FIX44b
BeginString_FIX44c
cvar.BeginString_FIX44d
cvar.BeginString_FIX44d
cvar.BeginString_FIX44b = "12"'\0'"45"
cvar.BeginString_FIX44b
cvar.BeginString_FIX44d
cvar.BeginString_FIX44e
BeginString_FIX44f

test_a("hello", "hi", "chello", "chi")

test_b("1234567", "hi")
